lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -9,7 +9,8 @@
 // first frame of the response.
 const BASE_RESPONSE_LATENCY = 16;
 const SCHEDULABLE_TASK_TITLE = 'TaskQueueManager::ProcessTaskFromWorkQueue';
-const SCHEDULABLE_TASK_TITLE_ALT = 'ThreadControllerImpl::DoWork';
+const SCHEDULABLE_TASK_TITLE_ALT1 = 'ThreadControllerImpl::DoWork';
+const SCHEDULABLE_TASK_TITLE_ALT2 = 'ThreadControllerImpl::RunTask';
```

Comment:
can you add a comment pointing to https://bugs.chromium.org/p/chromium/issues/detail?id=871204#c11 and indicating its in m69 and m70

Reply:
done